### PR TITLE
Exposing Nevermined Provenance capabilities 

### LINKED ISSUE #13: Support the Provenance API in CLI

### PR DESCRIPTION
**Pull request by aaitor**
## Description

Nevermined implements the W3C Provenance specs. This PR exposes the most relevant provenance functionalities in the CLI

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [X] Follows the code style of this project.
- [X] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

**Review by eruizgar91 — APPROVED** [review]
LGTM!

File: src/main/java/io/keyko/nevermined/cli/modules/utils/UtilsValidateSignature.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import io.keyko.nevermined.cli.helpers.CommandLineHelper;
+import io.keyko.nevermined.cli.models.CommandResult;
+import io.keyko.nevermined.cli.models.exceptions.CLIException;
+import picocli.CommandLine;
+
+import java.util.concurrent.Callable;
+
+@CommandLine.Command(
+        name = "validate-signature",
+        description = "It sign a message using the ECDSA credentials")
```
Comment (r-marques) [inline src/main/java/io/keyko/nevermined/cli/modules/utils/UtilsValidateSignature.java:14]: This description is wrong

File: src/test/java/io/keyko/nevermined/cli/UtilsIT.java [hunk 2]
```diff
@@ -20,5 +31,85 @@ public void utilsInfo() throws CLIException {
...
+        byte[] hashMessage= EthereumHelper.getEthereumMessageHash(message);
+        assertTrue(EthereumHelper.wasSignedByAddress(cli.getCredentials().getAddress(), signatureData, hashMessage));
+
+        String signatureString= EncodingHelper.signatureToString(signatureData);
+        System.out.println("Signature:" + signatureString);
+        Sign.SignatureData signatureGenerated =
+                EncodingHelper.stringToSignature(EthereumHelper.remove0x(signatureString));
+
+        assertTrue(EthereumHelper.wasSignedByAddress(cli.getCredentials().getAddress(), signatureGenerated, hashMessage));
+/*
```
Comment (r-marques) [inline src/test/java/io/keyko/nevermined/cli/UtilsIT.java:94]: Should this be removed?

**Review by r-marques — APPROVED** [review]
lgtm

**Pull request merged**